ibis/backends/duckdb/tests/test_register.py

```diff
@@ -234,9 +238,6 @@ def test_read_sqlite(con, tmp_path):
     ft = con.read_sqlite(path, table_name="t")
     assert ft.count().execute()
 
-    with pytest.raises(ValueError):
-        con.read_sqlite(path)
-
```

Comment:
Does this no longer raise?

Reply:
I should have put a note, this is already tested in a separate test, I thought it was redundant see

https://github.com/ibis-project/ibis/blob/9cb4c8d17f9d7960c216b324a0153b9c785ea09f/ibis/backends/duckdb/tests/test_register.py#L241-L249

Reply:
Ahh, gotcha.  thanks!